app/models/deploy.rb

```diff
@@ -110,7 +110,7 @@ def self.finished_naturally
   end
 
   def self.prior_to(deploy)
-    deploy.persisted? ? where("#{table_name}.id < ?", deploy.id) : self
+    deploy.persisted? ? where("#{table_name}.id < ?", deploy.id) : all
```

Comment:
Ah yes, good point. It appears that `all` is not deprecated, and now returns an arel:

```
irb(main):002:0> Deploy.where('id > 5').all.class
=> Deploy::ActiveRecord_Relation
```

and it's still listed in the API docs: http://api.rubyonrails.org/classes/ActiveRecord/Scoping/Named/ClassMethods.html#method-i-all